compio-runtime/src/attacher.rs

```diff
@@ -15,32 +21,30 @@ use crate::Runtime;
 /// itself is Send & Sync. We mark it !Send & !Sync to warn users, making them
 /// ensure that they are using it in the correct thread.
 #[derive(Debug, Clone)]
-pub struct Attacher {
+pub struct Attacher<S> {
+    source: S,
     // Make it thread safe.
     once: OnceLock<usize>,
-    // Make it !Send & !Sync.
-    _p: PhantomData<*mut ()>,
 }
 
-impl Attacher {
+impl<S> Attacher<S> {
     /// Create [`Attacher`].
-    pub const fn new() -> Self {
```

Comment:
Why is this not const anymore?

Reply:
I thought it couldn't be const, but I was wrong:(